### DISCUSSION TITLE
GLIBC_2.3x version not found

### DISCUSSION THREAD

**Discussion by vishalgattani** [jonasthewolf/gsn2x#333] [Q&A] [answered]
When I run the command `./gsn2x-Linux output.yaml` in the terminal, i get the following error. I have made the pre-built binary as an executable using `chmod +x gsn2x-Linux`. It works on my macOS using the mac binary, but on Ubuntu 20.04 it fails. Does anyone happen to know a workaround?

```shell
./gsn2x-Linux: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./gsn2x-Linux)
./gsn2x-Linux: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./gsn2x-Linux)
./gsn2x-Linux: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./gsn2x-Linux)
```

**Comment by jonasthewolf** [comment, marked as answer]
Hi @vishalgattani ,

the pre-compiled binaries are built on docker images. The Linux version uses Ubuntu latest, which I guess is currently 22.x.

I can try to reduce it to a lower version.

If you need it running immediately, you can clone the repo and run `cargo build --release`.
I currently suggest cloning the latest 2.8 version tag.

Best regards,

Jonas

**Reply by vishalgattani** [reply]
Thank you @jonasthewolf ! That worked. 

Best regards,
Vishal
